cmd/podman/exists.go

```diff
@@ -65,7 +65,7 @@ var (
 			podExistsCommand.GlobalFlags = MainGlobalOpts
 			return podExistsCmd(&podExistsCommand)
 		},
-		Example: `podman pod exists podID`,
+		Example: `podman pod exists mypod || podman pod create --name mypod`,
 	}
```

Comment:
I'd suggest keeping the original examples here for these three exists commands and adding the ones that you've worked up as the second example for each.  FW

Reply:
That's easy to do; but first, @baude, could you provide a little more guidance on your review comment of #2456? What sort of examples were you hoping to see?